roboat_planning:
  # Topics
  _pointCloudTopic: "velodyne_points"

  # Occupancy Map Params
  _mapResolution: 0.05
  _occuMapInflation: 0.0  #内层 实际障碍
  _occuMapField: 0.2  #外层 代价

  # Laser scan params
  _sensorRangeLimitMin: 0
  _sensorRangeLimitMax: 10.0 
  _sensorHeightLimitUpper: 0.5
  _sensorHeightLimitDown: -0.2
  _sensorCloudTimeout: 0.5

  # Alternative Path Params
  _samplingTipMargin: 0.1 # 1.0  0.5  延长距离  0.5 
  _samplingOutMargin: 0.1  # 1.0 平滑度
  _pathResolution: 0.2 #0.1一条path中的re  0.2 11
  _maxPathDistance: 15.0   #搜索长度
  _rollOutDensity: 0.2 #每条path 的dis 0.2 11
  _rollOutNumber: 30 #num  path

  _is_use_rspath: false
  _is_use_sidemodel: false
  _use_backwardmodel: false